Spielregeln.md

```diff
@@ -0,0 +1,88 @@
+# Spielregeln
+
+## Vorwort
+
+Bierball ist ein Sport, in welchem der Spaß aller Spielenden im Vordergrund steht. 
+Alle Teilnehmenden sind zu jeder Zeit mit Respekt zu behandeln.
+Von allen Teilnehmenden wird Fair-Play erwartet.
+
+## § 1 Spielziel
+
+(1) Das Spiel gilt als beendet, sobald ein Team die Getränke komplett geleert hat.
+
+(2) Ein Getränk gilt als geleert, wenn sich das Trinkbehältnis vom Spielenden mit Öffnung nach unten über den Kopf gehalten wurde, ohne dass eine nennenswerte Menge an Flüssigkeit aus dem Behältnis entweicht.
+Sollte zu viel Flüssigkeit entweichen, kann dies mit einem Strafschluck (siehe § 6 (1)) sanktioniert werden.
+
+## § 2 Teams
+
+(1) Gespielt wird mit zwei gleich großen Teams, die jeweils aus mindestens einer Person bestehen.
+
+## § 3 Spielgeräte
+
+(1) Die folgenden Spielgeräte werden zum Spielen benötigt:
+
+1. Ein Wurfgerät
+2. Ein Ziel
+3. Ein Getränkebehältnis pro spielender Person
+
+(2) Das Wurfgerät ist ein annähernd runder Gegenstand, der sich leicht einhändig werfen lässt.
```

Comment:
Es kam die Idee auf, die Anleitung für den "originalen" Ball irgendwo niederzuschreiben. Wäre hier oder eher ein eigenes Dokument eine passende Stelle?

Reply:
Ich glaube, dafür wäre ein anderes Dokument sinnvoller. Ich würde mich in den Regeln nicht so sehr auf diesen Ball festlegen wollen. Man könnte hier dann höchstens auf das andere Dokument verweisen.
Feel free, so eine Anleitung zu schreiben und dann nen PR aufs Repo zu stellen. (Bzw. kann ich dir auch einfach Zugriff aufs Repo geben, wenn du willst)

Reply:
Dann würde ich mich über einen Zugriff auf das Repo sehr freuen :D